File: dev-docs/snippets/invitations/create.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Create an app-level invitation. Only `email` is required.
export async function create(client: JsBaoClient, email: string) {
  // #region example
  const invitation = await client.invitations.create({
    email,
    role: "member",
    sendEmail: true,
  });
  // #endregion example
  return invitation.invitationId;
}

File: dev-docs/snippets/invitations/create.swift
import JsBaoClient

// Create an app-level invitation. Only `email` is required. Returns a typed
// `AppInvitationInfo`.
func create(client: JsBaoClient, email: String) async throws {
  // #region example
  let invitation = try await client.invitations.create(
    params: CreateInvitationParams(email: email, role: "member", sendEmail: true)
  )
  let invitationId = invitation.invitationId
  // #endregion example
  _ = invitationId
}
